x-pack/legacy/plugins/apm/public/new-platform/plugin.tsx

```diff
@@ -59,8 +63,17 @@ export interface ApmPluginStartDeps {
   data: DataPublicPluginStart;
 }
 
-const PluginsContext = createContext({} as ApmPluginStartDeps);
+export interface ConfigSchema {
+  apmIndexPatternTitle: string;
+  apmServiceMapEnabled: boolean;
+  apmUiEnabled: boolean;
+}
 
+// These are to be used until we switch over all our context handling to
+// kibana_react
+export const PluginsContext = createContext<
```

Comment:
What about moving this to the [context folder](https://github.com/elastic/kibana/tree/cd109fa7c7593cf174b74474734ec8fabf20d0f5/x-pack/legacy/plugins/apm/public/context)?